spec/features/users/user_profile_spec.rb

```diff
@@ -181,4 +181,48 @@
       expect(current_path).to eq(account_history_path)
     end
   end
+
+  context 'allows verified user to see their information' do
+    let(:pii) { { ssn: '1234', dob: '1920-01-01' } }
+    context 'time between sign in and remember device' do
+      it 'does not have prompt to authenticate device' do
+        profile = create(:profile, :active, :verified, pii: pii)
+        sign_in_user(profile.user)
+        check t('forms.messages.remember_device')
+        fill_in_code_with_last_phone_otp
+        click_submit_default
+        visit account_path
+        expect(page).to_not have_link(t('account.re_verify.footer'))
+      end
+    end
+
+    context 'when time expired' do
+      before do
+        allow(IdentityConfig.store).to receive(:pii_lock_timeout_in_minutes).and_return(1)
+      end
+      it 'has a prompt to authenticate device' do
+        profile = create(:profile, :active, :verified, pii: pii)
+        user = profile.user
+        sign_in_user(user)
+
+        check t('forms.messages.remember_device')
+        fill_in_code_with_last_phone_otp
+        click_submit_default
+
+        timeout_in_minutes = IdentityConfig.store.pii_lock_timeout_in_minutes.to_i
+        travel_to((timeout_in_minutes + 26).minutes.from_now) do
```

Comment:
It seems like we shouldn't need to stub the value if we're referencing it in the `travel_to`. Is there something special about the 26, or could we offset by the minimal amount (e.g. 1) instead? Also, since `pii_lock_timeout_in_minutes` is already numeric, shouldn't need to cast it I think.

```suggestion
      it 'has a prompt to authenticate device' do
        profile = create(:profile, :active, :verified, pii: pii)
        user = profile.user
        sign_in_user(user)

        check t('forms.messages.remember_device')
        fill_in_code_with_last_phone_otp
        click_submit_default

        timeout_in_minutes = IdentityConfig.store.pii_lock_timeout_in_minutes
        travel_to((timeout_in_minutes + 1).minutes.from_now) do
```